Tools/Cli-LoRa-Device-Provisioning/Keygen.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Cli_LoRa_Device_Provisioning
+{
+    using System;
+
+    public static class Keygen
+    {
+        public static string Generate(int keyLength)
+        {
+            var randomKey = string.Empty;
```

Comment:
probably a good idea to use a StringBuilder.

Reply:
done!